web/src/components/Surfaces/ImageCard.tsx

```diff
@@ -48,25 +48,32 @@ export default function ImageCard({
   return (
     <>
       <Card
-        className={`group relative aspect-square cursor-pointer select-none overflow-hidden bg-slate-500 p-6 
-      transition-all duration-200 ease-out hover:shadow-2xl sm:hover:scale-[103%] ${className}`}
+        tabIndex={1}
+        className={`group relative aspect-square cursor-pointer select-none overflow-hidden transition-all duration-200 ease-out 
+        hover:shadow-2xl focus-visible:scale-90 focus-visible:ring-4 focus-visible:ring-indigo-800 focus-visible:ring-offset-4 sm:hover:scale-[103%] ${className}`}
         onClick={handleDialogOpen}
+        onKeyUp={(e) => {
+          if (e.key === "Enter") {
+            handleDialogOpen();
+          }
+        }}
```

Comment:
> I typically use `onKeyDown` for this - any reason for using `onKeyUp`?

onkeyup is more or less equivalent to the deprecated onKeyPress. I use it because it ensures that the key is fully pressed before activating (and possibly prevents it from activating many times since the OS repeats key pressed when you hold it down)

Reply:
Sounds good